.github/workflows/compatiblity_test_on_schedule.yml

```diff
@@ -10,6 +10,7 @@ jobs:
   matrix_preparation:
     name: Prepare Container List
     runs-on: ubuntu-latest
+    if: ${{github.event_name != 'schedule' || github.repository == 'hpcaitech-colossalai'}}
```

Comment:
Should the repo be `hpcaitech/ColossalAI`? And why not run when the event name is `schedule`?